image/imageimagick.go

```diff
@@ -0,0 +1,73 @@
+// +build imagick
+
+package image
+
+import (
+	"gopkg.in/gographics/imagick.v2/imagick"
+	"strconv"
+)
+
+// Image type for holding configuration
+type Image struct {
+	ImageDir             string  `long:"imagedir" description:"Directory where generated images are placed"`
+	ImageURLDir          string  `long:"imageurldir" description:"The directory part of the URL where browsers will find the images"`
+	ImageFile            string  `long:"imagefile" description:"Full path to the image that will be served"`
+	ImageTextBeforeAmt   string  `long:"imagetextbeforeamt" description:"The text on the first line before the amount"`
+	ImageTextAfterAmt    string  `long:"imagetextafteramt" description:"The text on the first line after the amount"`
+	ImageTextSecondLine  string  `long:"imagetextsecondline" description:"The text on the second line"`
```

Comment:
What about having those three `ImageText` values in one config option and replacing for example `${amt}` with the amt and making a new line on `\n`?

Reply:
Fixed. Now only one text with possibility to add line feeds and {Amount}.